File: manager/urls.py
# -*- coding: utf-8 -*-

from django.conf.urls.defaults import *

urlpatterns = patterns(
    'manager.views',
    url(r'^$', 'index'),
    url(r'^add_event/$', 'ajax_add_event', name='ajax_add_event'),
    url(r'^add_event/(?P<pk>\d+)/$', 'ajax_add_event'),
    url(r'^del_event/$', 'ajax_del_event', name='ajax_del_event'),
    url(r'^change_date/$', 'ajax_change_date', name='ajax_change_date'),
    url(r'^get_events/$', 'ajax_get_events', name='ajax_get_events'),
    url(r'^ajax_get_user_courses/$', 'ajax_get_user_courses', name='ajax_get_user_courses'),
    url(r'^get_course_tree/$', 'ajax_get_course_tree', name='ajax_get_course_tree'),
    url(r'^get_rooms/$', 'ajax_get_rooms', name='ajax_get_rooms'),
    url(r'^del_user_course/$', 'ajax_del_user_couse', name='ajax_del_user_couse'),
    url(r'^get_coach_list/$', 'ajax_get_coach_list', name='ajax_get_coach_list'),
    url(r'^get_unstatus_event/$', 'ajax_get_unstatus_event', name='ajax_get_unstatus_event'),
    url(r'^save_event_status/$', 'ajax_save_event_status', name='ajax_save_event_status'),
    url(r'^get_status_timer/$', 'ajax_get_status_timer', name='ajax_get_status_timer'),
    url(r'^copy_week/$', 'ajax_copy_week', name='ajax_copy_week'),
    url(r'^add_user_course/$', 'ajax_add_user_course', name='ajax_add_user_course')
    )

urlpatterns += patterns(
    'manager.ajax',
    url(r'^get_user_info/$', 'get_user_info'),
    url(r'^get_users_info_by_name/$', 'get_users_info_by_name'),
    url(r'^set_user_info/$', 'set_user_info'),
    url(r'^available_courses/$', 'available_courses'),
    )

File: air/js/URLS.js
if (typeof(URLS) == 'undefined') URLS = {};
var root = 'http://127.0.0.1:8000';
URLS.update_url = root+'/static/air/updater/update.xml';

URLS.get_course_tree = root+'/manager/get_course_tree/';
URLS.get_user_courses = root+'/manager/ajax_get_user_courses/';
URLS.add_event = root+'/manager/add_event/';
URLS.del_user_course = root+'/manager/del_user_course/';
URLS.add_user_course = root+'/manager/add_user_course/';
URLS.get_coach_list = root+'/manager/get_coach_list/';
URLS.get_unstatus_event = root+'/manager/get_unstatus_event/';
URLS.save_event_status = root+'/manager/save_event_status/';
URLS.get_status_timer = root+'/manager/get_status_timer/';
URLS.copy_week = root+'/manager/copy_week/';
URLS.del_event = root+'/manager/del_event/';
URLS.change_date = root+'/manager/change_date/';
URLS.get_events = root+'/manager/get_events/';
URLS.get_rooms = root+'/manager/get_rooms/';
URLS.get_user_data = root+'/ajax/rfid/';
